Failed tests (12558): TestChaos

The following test appears to have failed:

[#12558](https://circleci.com/gh/cockroachdb/cockroach/12558):

```
I0209 20:18:09.707434 1 server/node.go:466  node connected via gossip and verified as part of cluster "6c8525a6-1fe9-4e31-ba8b-db0ef56d2e38"
I0209 20:18:09.707774 1 server/node.go:301  Started node with [[ssd]=/data0.0] engine(s) and attributes []
I0209 20:18:09.707946 1 server/server.go:266  starting https server at roach0.local:26257
I0209 20:18:09.708620 1 sql/pgwire/server.go:94  starting postgres server at 172.17.0.13:15432
I0209 20:18:10.650553 1 gossip/gossip.go:933  starting client to roach1.local:26257
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0xb21f83]

goroutine 30 [running]:
github.com/cockroachdb/cockroach/sql.(*databaseCache).setID(0x0, 0xc8204f7012, 0x4, 0xc800000032)
	/go/src/github.com/cockroachdb/cockroach/sql/database.go:63 +0xd3
github.com/cockroachdb/cockroach/sql.(*planner).getDatabaseID(0xc82050e000, 0xc8204f7012, 0x4, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/sql/database.go:141 +0x227
github.com/cockroachdb/cockroach/sql.(*planner).getTableID(0xc82050e000, 0xc8204dc8d0, 0x4, 0x0)
	/go/src/github.com/cockroachdb/cockroach/sql/table.go:285 +0xea
github.com/cockroachdb/cockroach/sql.(*planner).getTableLease(0xc82050e000, 0xc8204dc8d0, 0x0, 0x7fbb4026cd90)
	/go/src/github.com/cockroachdb/cockroach/sql/table.go:250 +0x173
github.com/cockroachdb/cockroach/sql.(*planner).getAliasedTableLease(0xc82050e000, 0x7fbb4026cae8, 0xc8204d1980, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/sql/plan.go:246 +0x24f
--
github.com/cockroachdb/cockroach/sql/pgwire.(*Server).serve.func1(0xc8202e6400, 0x7fbb4026c578, 0xc8200f62f0)
	/go/src/github.com/cockroachdb/cockroach/sql/pgwire/server.go:126 +0x90
created by github.com/cockroachdb/cockroach/sql/pgwire.(*Server).serve
	/go/src/github.com/cockroachdb/cockroach/sql/pgwire/server.go:131 +0x20a

goroutine 1 [select]:
github.com/cockroachdb/cockroach/cli.runStart(0x1e4a9e0, 0xc820257a00, 0x0, 0x8, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/cli/start.go:142 +0x117b
github.com/spf13/cobra.(*Command).execute(0x1e4a9e0, 0xc820257980, 0x8, 0x8, 0x0, 0x0)
	/go/src/github.com/spf13/cobra/command.go:565 +0x63b
github.com/spf13/cobra.(*Command).ExecuteC(0x1e477e0, 0x1e4a9e0, 0x0, 0x0)
	/go/src/github.com/spf13/cobra/command.go:656 +0x56b
github.com/spf13/cobra.(*Command).Execute(0x1e477e0, 0x0, 0x0)
	/go/src/github.com/spf13/cobra/command.go:615 +0x2d
github.com/cockroachdb/cockroach/cli.Run(0xc82000a0b0, 0x9, 0x9, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/cli/cli.go:85 +0x7c
main.main()
	/go/src/github.com/cockroachdb/cockroach/main.go:37 +0xe6

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 8 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x21992a0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x64
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0xbc

goroutine 10 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 11 [select]:
github.com/cockroachdb/cockroach/rpc.(*RemoteClockMonitor).MonitorRemoteOffsets(0xc8202e4f60, 0xc8202645b0)
	/go/src/github.com/cockroachdb/cockroach/rpc/clock_offset.go:156 +0x742
github.com/cockroachdb/cockroach/server.NewServer.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:130 +0x31
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202e4f80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 12 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:213 +0x1b6
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202e5120)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 13 [select]:
github.com/cockroachdb/cockroach/util.NewFeed.func1()
	/go/src/github.com/cockroachdb/cockroach/util/feed.go:61 +0x157
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202ec1a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 14 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc820263f40)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:225 +0xca5
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:212 +0x20
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202ec200)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 15 [select]:
github.com/cockroachdb/cockroach/sql.(*LeaseManager).RefreshLeases.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/lease.go:601 +0xdd0
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202f00f0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 16 [select]:
github.com/cockroachdb/cockroach/sql.NewExecutor.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/executor.go:176 +0x134
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202f0270)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 34 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0ee8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8202d9db0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8202d9db0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc8202d9d50, 0x0, 0x7fbb4026c288, 0xc82059abc0)
	/usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*TCPListener).AcceptTCP(0xc820026490, 0xc800000000, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202f0570)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 35 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:72 +0x5b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202e53c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 36 [chan receive]:
github.com/cockroachdb/cockroach/gossip.(*server).start.func2()
	/go/src/github.com/cockroachdb/cockroach/gossip/server.go:262 +0x57
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202e5460)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 37 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).bootstrap.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:770 +0x207
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202e5480)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 38 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:806 +0x33a
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202e54a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 39 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc82026e0f0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x3a
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x490

goroutine 40 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1642 +0x85f
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202eda60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 41 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:582 +0x119
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041c200)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 42 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:642 +0x361
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041c220)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 43 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:661 +0x361
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041c240)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 31 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ddda0, 0x7fbb4026d200, 0xc8203ecd80, 0xc8202ee660)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:336 +0xa4
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ddda0, 0x7fbb4026d200, 0xc8203ecd80, 0xc8202ee660)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:243 +0x3f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:244 +0x89c

goroutine 22 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:352 +0x2ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef520)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 23 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:352 +0x2ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef540)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 24 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:352 +0x2ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef560)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 25 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:352 +0x2ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef580)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 26 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:352 +0x2ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef5a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 27 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:352 +0x2ee
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef5c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 28 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc8202eeea0, 0xece4c4082, 0x2a3679a4, 0x2198b20, 0xc8202e1a40, 0xc8202645b0, 0xc820268b60, 0xc8203eee00)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x53f
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc820268b60, 0xc8202f05d0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:214 +0x78
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc8202f0720, 0xc820150d40)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:223 +0x246
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:216 +0x1e1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8200ef5e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 46 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startPublishStatuses.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:511 +0x23e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041c920)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 47 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:479 +0x3f1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041c940)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 48 [select]:
github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
	/go/src/github.com/cockroachdb/cockroach/ts/db.go:76 +0x141
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8204825f0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 49 [select]:
github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
	/go/src/github.com/cockroachdb/cockroach/ts/db.go:76 +0x141
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc820482600)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 50 [select]:
github.com/cockroachdb/cockroach/server.(*Server).startWriteSummaries.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:302 +0x169
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc820482610)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 51 [select]:
github.com/cockroachdb/cockroach/sql.(*SchemaChangeManager).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/schema_changer.go:487 +0x16fc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041cb60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 52 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0e28, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820401020, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820401020, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc820400fc0, 0x0, 0x7fbb4026c288, 0xc8200ef7a0)
	/usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*TCPListener).AcceptTCP(0xc820026660, 0xc82047bec8, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82041ce20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 53 [chan receive]:
github.com/cockroachdb/cockroach/sql/pgwire.(*Server).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/sql/pgwire/server.go:91 +0x4a
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc820482ae0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 54 [select, locked to thread]:
runtime.gopark(0x17921d0, 0xc820025f28, 0x1592f48, 0x6, 0xc820251318, 0x2)
	/usr/local/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc820025f28, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc820025f28)
	/usr/local/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal1_unix.go:227 +0x353
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 133 [select]:
github.com/cockroachdb/cockroach/rpc.(*Client).runHeartbeat(0xc820118e00, 0x3b9aca00, 0x6fc23ac00, 0x4000000000000000, 0x0, 0x0, 0xc8202513e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:347 +0x77b
github.com/cockroachdb/cockroach/rpc.NewClient.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:167 +0x36
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82058b340)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 32 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0ca8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820401100, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820401100, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8204010a0, 0xc820253800, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200266a0, 0xc820253800, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ddda0, 0x7fbb4026c600, 0xc820235600, 0x7fbb4026d200, 0xc8203ecd80, 0xc8202ec710, 0xc8202ee660)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:246 +0x5d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0x90c

goroutine 61 [select]:
github.com/cockroachdb/cockroach/gossip.(*client).gossip(0xc8202da5f0, 0xc8202545a0, 0xc8202645b0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/gossip/client.go:242 +0x80f
github.com/cockroachdb/cockroach/gossip.(*client).start.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/client.go:73 +0x162
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8202f1860)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 66 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ddda0, 0x7fbb4026d200, 0xc82023a180, 0xc82000e3c0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:336 +0xa4
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ddda0, 0x7fbb4026d200, 0xc82023a180, 0xc82000e3c0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:243 +0x3f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:244 +0x89c

goroutine 67 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0be8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820400990, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820400990, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820400930, 0xc820252400, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200f60d8, 0xc820252400, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ddda0, 0x7fbb4026c600, 0xc8200a6000, 0x7fbb4026d200, 0xc82023a180, 0xc8202ed1b0, 0xc82000e3c0)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:246 +0x5d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0x90c

goroutine 62 [select]:
github.com/cockroachdb/cockroach/rpc.(*Client).runHeartbeat(0xc82009c000, 0x3b9aca00, 0x6fc23ac00, 0x4000000000000000, 0x0, 0x0, 0xc8202513e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:347 +0x77b
github.com/cockroachdb/cockroach/rpc.NewClient.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:167 +0x36
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8204845c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 98 [IO wait]:
net.runtime_pollWait(0x7fbb42ad08e8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc82054c6f0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82054c6f0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82054c690, 0xc82049f800, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820026278, 0xc82049f800, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/rpc.(*Client).input(0xc82021d320)
	/usr/local/go/src/net/rpc/client.go:109 +0xbf
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0xd2

goroutine 65 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0b28, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8201f4220, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8201f4220, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8201f41c0, 0xc820095000, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820026150, 0xc820095000, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/rpc.(*Client).input(0xc8202dc660)
	/usr/local/go/src/net/rpc/client.go:109 +0xbf
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0xd2

goroutine 86 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ddda0, 0x7fbb4026d200, 0xc820200a80, 0xc820545260)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:336 +0xa4
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ddda0, 0x7fbb4026d200, 0xc820200a80, 0xc820545260)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:243 +0x3f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:244 +0x89c

goroutine 79 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ddda0, 0x7fbb4026d200, 0xc820001800, 0xc82053b920)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:336 +0xa4
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ddda0, 0x7fbb4026d200, 0xc820001800, 0xc82053b920)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:243 +0x3f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:244 +0x89c

goroutine 138 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8202ddda0, 0x7fbb4026d200, 0xc820201800, 0xc82053af00)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:336 +0xa4
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8202ddda0, 0x7fbb4026d200, 0xc820201800, 0xc82053af00)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:243 +0x3f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:244 +0x89c

goroutine 80 [IO wait]:
net.runtime_pollWait(0x7fbb42ad09a8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820053090, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820053090, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820053030, 0xc820232000, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200f6378, 0xc820232000, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ddda0, 0x7fbb4026c600, 0xc8200a6b00, 0x7fbb4026d200, 0xc820001800, 0xc820261080, 0xc82053b920)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:246 +0x5d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0x90c

goroutine 87 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0a68, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820053020, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820053020, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820052fc0, 0xc82049ec00, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200f6360, 0xc82049ec00, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ddda0, 0x7fbb4026c600, 0xc8200a62c0, 0x7fbb4026d200, 0xc820200a80, 0xc8201e67d0, 0xc820545260)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:246 +0x5d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0x90c

goroutine 91 [select]:
github.com/cockroachdb/cockroach/server.(*rpcTransport).processQueue(0xc8202f0090, 0x200000002)
	/go/src/github.com/cockroachdb/cockroach/server/raft_transport.go:163 +0xd79
created by github.com/cockroachdb/cockroach/server.(*rpcTransport).Send
	/go/src/github.com/cockroachdb/cockroach/server/raft_transport.go:196 +0x150

goroutine 92 [select]:
github.com/cockroachdb/cockroach/rpc.(*Client).runHeartbeat(0xc82009c700, 0x3b9aca00, 0x6fc23ac00, 0x4000000000000000, 0x0, 0x0, 0xc8202513e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:347 +0x77b
github.com/cockroachdb/cockroach/rpc.NewClient.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:167 +0x36
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc8204d0840)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 134 [select]:
github.com/cockroachdb/cockroach/rpc.(*Client).runHeartbeat(0xc820118ee0, 0x3b9aca00, 0x6fc23ac00, 0x4000000000000000, 0x0, 0x0, 0xc8202513e0)
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:347 +0x77b
github.com/cockroachdb/cockroach/rpc.NewClient.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:167 +0x36
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202645b0, 0xc82058b380)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x62

goroutine 140 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0828, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820584c30, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820584c30, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820584bd0, 0xc82056c800, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820026488, 0xc82056c800, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/rpc.(*Client).input(0xc82053b020)
	/usr/local/go/src/net/rpc/client.go:109 +0xbf
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0xd2

goroutine 139 [IO wait]:
net.runtime_pollWait(0x7fbb42ad0768, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820584ca0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820584ca0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820584c40, 0xc820233c00, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200264b0, 0xc820233c00, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8202ddda0, 0x7fbb4026c600, 0xc820234580, 0x7fbb4026d200, 0xc820201800, 0xc8204ec810, 0xc82053af00)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:246 +0x5d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:249 +0x90c

goroutine 160 [IO wait]:
net.runtime_pollWait(0x7fbb42ad06a8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8205cf640, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8205cf640, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8205cf5e0, 0xc8205b0800, 0x400, 0x400, 0x0, 0x7fbb42ac3050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200264b8, 0xc8205b0800, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
E0209 20:18:14.233414 27474 acceptance/cluster/localcluster.go:410  node=1 status=restart
I0209 20:18:14.240911 27474 acceptance/chaos_test.go:162  client 1 shutting down
I0209 20:18:20.519860 27474 acceptance/chaos_test.go:162  client 2 shutting down
I0209 20:18:20.522910 27474 acceptance/cluster/localcluster.go:538  stopping
I0209 20:18:20.963159 27474 acceptance/cluster/localcluster.go:577  node 0: stderr at /tmp/circle-artifacts.WEZJvvO/acceptance/TestChaos/roach0.local/stderr.2016-02-09T20_18_20Z.log
--- FAIL: TestChaos (24.32s)
	chaos_test.go:238: read tcp 127.0.0.1:46284->127.0.0.1:32783: read: connection reset by peer
	chaos_test.go:280: dial tcp 0.0.0.0:32783: getsockopt: connection refused
	localcluster.go:510: expected event &{1 die}, got &{0 die} (after [])
=== RUN   TestGossipPeerings
I0209 20:18:23.116550 27474 acceptance/cluster/localcluster.go:219  started docker-spy: 172.17.0.15
I0209 20:18:23.116578 27474 acceptance/cluster/localcluster.go:226  initializing cluster
I0209 20:18:24.943481 27474 acceptance/cluster/localcluster.go:333  creating ca (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.893484475
I0209 20:18:25.044091 27474 acceptance/cluster/localcluster.go:338  creating node (1024bit) certs in: /home/ubuntu/cockroach/.localcluster.certs.893484475
I0209 20:18:26.756876 27474 acceptance/cluster/localcluster.go:389  *** started roach0.local ***
  ui:    https://0.0.0.0:32787
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:32: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	main_test.go:62: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/sql/database.go
+++ b/sql/database.go
@@ -60,6 +60,9 @@ func (s *databaseCache) getID(name string) ID {
 }
 
 func (s *databaseCache) setID(name string, id ID) {
+	if s == nil {
+		return
+	}
 	s.mu.Lock()
 	defer s.mu.Unlock()
 	s.databases[name] = id

